website/source/docs/commands/acl/token-self.html.md.erb

```diff
@@ -0,0 +1,41 @@
+---
+layout: "docs"
+page_title: "Commands: acl token self"
+sidebar_current: "docs-commands-acl-token-self"
+description: >
+  The token self command is used to fetch information about the currently set
+  ACL token.
+---
+
+# Command: acl token self
+
+The `acl token self` command is used to fetch information about the currently set ACL token.
+
+## Usage
+
+```
+nomad acl token self
+```
+
+## General Options
+
+<%= partial "docs/commands/_general_options" %>
+
+## Examples
+
+Fetch information about an existing ACL token:
+
+```
+$ export NOMAD_TOKEN=d532c40a-30f1-695c-19e5-c35b882b0efd
+
+$ nomad acl tokenjself
+Accessor ID  = d532c40a-30f1-695c-19e5-c35b882b0efd
+Secret ID    = 85310d07-9afa-ef53-0933-0c043cd673c7
+Name         = my token
+Type         = client
+Global       = false
+Policies     = [foo bar]
```

Comment:
[nitpick] I prefer CSVs to Go's syntax for output, but whatever is consistent with other output is fine by me.